README.rst

```diff
@@ -66,10 +66,30 @@ pkgdb2::
 
     (my-pkgdb2-env)$ python createdb.py
 
+Setting up PostgreSQL
+=====================
+You can set up PostgreSQL instead of sqlite
```

Comment:
Having the data dump can make things easier, but sometime for quick testing the sqlite db can be enough.

I think we should just add a couple of sentences here explaining why one would want to use postgresql or sqlite.